docs/dev/android/setup-android-build-environment.md

```diff
@@ -93,7 +79,10 @@ environment variables mentioned above are in place.
    You can see a list of projects by executing `./gradlew projects` and a list
    of tasks by executing `./gradlew tasks`.
 
-### Publishing the components to your local maven repository.
+The above can be skipped if using `Android Studio`, as the project directory can be imported
+and all the build details can be left to the IDE.
+
+### Publishing the components to your local maven repository
```

Comment:
While we're at it: should we add a note that this is incomplete? (We didn't setup/test publishing to maven)

Reply:
I vote for removing this version, since we didn't test it. We can always look it up later.